README.adoc

```diff
@@ -140,16 +140,15 @@ To link this JSON file to your application, create a `start/src/main/java/io/ope
 include::finish/src/main/java/io/openliberty/guides/config/CustomConfigSource.java[tags=customConfig]
 ----
 
-The `setProperties()` private method reads the key value pairs from the `CustomConfigSource.json` JSON file and writes the information into a map.
+The `getProperties()` method reads the key value pairs from the `resources/CustomConfigSource.json` JSON file and writes the information into a map.
 
-To register the custom configuration source, create a `start/src/main/resources/META-INF/services/org.eclipse.microprofile.config.spi.ConfigSource` file and add the full class name into it:
+Lastly, it is an important step to register the custom configuration source, create a file called `org.eclipse.microprofile.config.spi.ConfigSource` under `start/src/main/resources/META-INF/services/` directory and add the full class name into it:
```

Comment:
The `org.eclipse.microprofile.config.spi.ConfigSource` is the file name, so it is very misleading if you put it in the conventional way, that's why I need to emphasize the file is called `org.eclipse.microprofile.config.spi.ConfigSource`. Otherwise, the user may think `ConfigSource` is the file name.

Reply:
I still think "create the config source in the src/main/resources/META-INF/services/org.eclipse.microprofile.config.spi.ConfigSource file:" is clear enough to me. Plus this way, user don't have to copy the path and file name separately.

Reply:
But it's not a critical issue. I'm ok the way you have it. Just don't need "and add the full class name into it:" right?

Reply:
`the full class name` refers to the content of that file, not the name of the file.